src/main/java/uk/gov/mca/beacons/service/mappers/BeaconMapper.java

```diff
@@ -44,22 +45,36 @@ public Beacon fromDTO(BeaconDTO dto) {
 
     beacon.setBeaconStatus(getStatusOrNull("status", attributes));
 
-    beacon.setCreatedDate(getDateOrNull("createdDate", attributes));
+    beacon.setCreatedDate(getDateTimeOrNull("createdDate", attributes));
     beacon.setBatteryExpiryDate(getDateOrNull("batteryExpiryDate", attributes));
     beacon.setLastServicedDate(getDateOrNull("lastServicedDate", attributes));
 
     return beacon;
   }
 
-  private static LocalDateTime getDateOrNull(
+  private static LocalDateTime getDateTimeOrNull(
     String key,
     Map<String, Object> attributes
   ) {
     final var attributeValue = attributes.get(key);
     if (attributeValue == null) return null;
 
-    final var result = LocalDateTime.parse((String) attributeValue);
-    return result;
+    return LocalDateTime.parse((String) attributeValue);
+  }
+
+  private static LocalDate getDateOrNull(
+    String key,
+    Map<String, Object> attributes
+  ) {
+    final var attributeValue = (String) attributes.get(key);
+    if (attributeValue == null) return null;
+
+    return LocalDate.parse(removeTime(attributeValue));
+  }
+
+  private static String removeTime(String dateTimeString) {
+    final var isoDateStringLength = 10; // YYYY-MMM-DD
```

Comment:
A typo here, I assume this should read: `YYYY-MM-DD`?  You've added an extra month